NEWS.md

```diff
@@ -1,5 +1,11 @@
 # ggplot2 (development version)
 
+* `scale_*_manual()` no longer displays extra legend keys, or changes their 
+  order, when a named `values` argument has more items than the data. The 
+  previous behaviour can be replicated by using 
+  `scale_*_manual(values = vals, limits = names(vals))`. (@teunbrand, @banfai, 
+  #4511, #4534, #4556, #4545, #4616)
```

Comment:
While there are many issues filed, I think they can be summarised to these two ones.

```suggestion
  #4511, #4534)
```